keps/sig-storage/20180515-svcacct-token-volumes.md

```diff
@@ -159,6 +161,55 @@ sources:
     audience: ca.istio.io
 ```
 
+### File Permission
+
+The secret projections are currently written with world readable (0644) file
+permissions. Given that file permissions are one of the oldest and most hardened
+isolation mechanisms on unix, this is not ideal. We would like to
+opportunistically restrict permissions for projected service account tokens as
+long we can show that they won’t break users if we are to migrate away from
+secrets to distribute service account credentials.
+
+#### Proposed heuristics
+
++   *Case 1*: The pod has an fsGroup set. We can set the file permission on the
+    token file to 0600 and let the fsGroup mechanism work as designed. It will
+    set the permissions to 0640, chown the token file to the fsGroup and start
+    the containers with a supplemental group that grants them access to the
+    token file. This works today.
```

Comment:
cc @gnufied to make sure the new fsgroup change policy feature still supports this